packages/devtools_app/lib/src/memory/memory_chart.dart

```diff
@@ -51,11 +51,6 @@ class MemoryChartState extends State<MemoryChart> with AutoDisposeMixin {
 
   @override
   void initState() {
-    _setupDartChartController();
```

Comment:
you cannot load the theme in initState() so this setup has to wait for when dependencies change.
The charts look fine so I think everything is alright. It would be much better if the controllers didn't define the colors when we shift this code over to use our own custom charting solution.